helm/distributed-idol/templates/content/statefulset.yaml

```diff
@@ -1,109 +1,48 @@
-        app: {{ $component.name }}
     spec:
-      imagePullSecrets:
-        {{- range .Values.global.imagePullSecrets }}
-        - name: {{ . }}
-        {{- end }}
       terminationGracePeriodSeconds: 300
-      containers:
-      - name: {{ $component.name }}
-        image: {{ include "idol-library.idolImage" (dict "root" . "idolImage" $component.idolImage )}}
-        imagePullPolicy: IfNotPresent
-        command:
-        - "/bin/bash"
-        args: 
-        - "-c"
-        - |
-{{ tpl (.Files.Get "resources/content_startup.sh") . | indent 10 }}
-        volumeMounts:
-        - name: config-map
-          mountPath: /etc/config/idol
-        - name: index
-          mountPath: /opt/idol/content/index
-{{- if .Values.setupMirrored }}
-        - name: archive-share
-          mountPath: /opt/idol/archive
-{{- end }}
-        {{- range $component.additionalVolumeMounts }}
-        - {{ . | toYaml | nindent 10 }}
-        {{- end }}
-        env:
-        - name: IDOL_CONTENT_SERVICE_PORT_ACI_PORT
-          # This doesn't seem to be set by our headless service
-          value: {{ $component.aciPort | quote }}
-        - name: IDOL_COMMAND_PARAMS
-          value: "-idolcluster"
-        {{- if $component.envConfigMap }}
-        envFrom:
-        - configMapRef: {{ $component.envConfigMap | quote }}
-        {{ end }}
-        lifecycle:
-          postStart:
-            exec:
-              command:
-              - /bin/bash
-              - "-c"
-              - |
-{{ tpl (.Files.Get "resources/content_postStart.sh") . | indent 16 }}
-          preStop:
-            exec:
-              command:
-              - /bin/bash
-              - "-c"
-              - |
-{{ tpl (.Files.Get "resources/content_preStop.sh") . | indent 16 }}
-        livenessProbe:
-          httpGet:
-            path: /a=getpid
-            port: {{ $component.aciPort | int }}
-{{- template "idol-library.standardLivenessProbe" $component.livenessProbe }}
-        ports:
-        - containerPort: {{ $component.aciPort | int }}
-          name: aci-port
-          protocol: TCP
-        - containerPort: {{ $component.servicePort | int }}
-          name: service-port
-          protocol: TCP
-        - containerPort: {{ $component.indexPort | int }}
-          name: index-port
-          protocol: TCP
-      volumes:
-      - name: config-map
-        configMap:
-          name: {{ default (printf "%s-default-cfg" $component.name) $component.existingConfigMap }}
-{{- if .Values.setupMirrored }}
-      - name: archive-share
-        persistentVolumeClaim:
-          claimName: backup-archive-pvc
-{{- end }}
-      {{- range $component.additionalVolumes }}
-      - {{ . | toYaml | nindent 8 }}
-      {{- end }}
   volumeClaimTemplates:
   - metadata:
       name: index
-      labels: {{- include "distributedidol.labels" . | nindent 8 }}
+      labels: {{- include "distributedidol.labels" $root | nindent 8 }}
```

Comment:
This should use `idol-library.labels.volumeClaimTemplates` from #81

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/80/commits/1a2598c339b3818b8da53f103c0b190de7c29687